app/javascript/components/create-nuage-cloud-subnet-form.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import NuageCloudSubnetForm from './forms/nuage-cloud-subnet-form'
+import { createSubnet, fetchRouter } from '../utils/api.js'
+
+class CreateNuageCloudSubnetForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleFormStateUpdate = this.handleFormStateUpdate.bind(this);
+    this.state = {
+      emsId: false,
```

Comment:
Ok now i don't like it either :D Now when i see it i would expect that those values will be bool in the future which is not true.  I think it would be best just keep them undefined until you initialize them.